neusomatic/python/generate_dataset.py

```diff
@@ -849,116 +849,176 @@ def find_records(input_record):
+                                record[3] == curren_pos_records[0][3] and record[4] == curren_pos_records[0][4]):
                             curren_pos_records.append(record)
                         else:
                             emit_flag = True
                     else:
                         curren_pos_records.append(record)
 
-                if emit_flag:
-                    if curren_pos_records:
-                        rrs = []
-                        for record_ in curren_pos_records:
-                            chrom, pos, ref, alt = [str(record_[0]), int(
-                                record_[1]), record_[3], record_[4]]
-                            r_ = []
-                            if len(ref) == len(alt) and len(ref) > 1:
-                                for ii in range(len(ref)):
-                                    ref_ = ref[ii]
-                                    alt_ = alt[ii]
-                                    if ref_ != alt_:
-                                        r_.append(
-                                            [chrom, pos + ii, ref_, alt_])
-                            else:
-                                r_ = [[chrom, pos, ref, alt]]
-
-                            ann = [0] * 93
-                            if record_[1] == record_[11]:
-                                if record_[3] == record_[13] and record_[4] == record_[14]:
-                                    ann = record_[15:]
-                                elif len(record_[3]) > len(record_[4]) and len(record_[13]) > len(record_[14]) and \
-                                        (record_[4]) == (record_[14]):
-                                    if (len(record_[3]) > len(record_[13]) and record_[3][0:len(record_[13])] == record_[13]) or (
-                                            len(record_[13]) > len(record_[3]) and record_[13][0:len(record_[3])] == record_[3]):
+                    if emit_flag:
+                        if curren_pos_records:
+                            rrs = []
+                            for record_ in curren_pos_records:
+                                chrom, pos, ref, alt = [str(record_[0]), int(
+                                    record_[1]), record_[3], record_[4]]
+                                r_ = []
+                                if len(ref) == len(alt) and len(ref) > 1:
+                                    for ii in range(len(ref)):
+                                        ref_ = ref[ii]
+                                        alt_ = alt[ii]
+                                        if ref_ != alt_:
+                                            r_.append(
+                                                [chrom, pos + ii, ref_, alt_])
+                                else:
+                                    r_ = [[chrom, pos, ref, alt]]
+
+                                ann = [0] * 93
```

Comment:
Having a constant declared which captures the number of columns makes the code more readable.